site/content/resources/support.md

```diff
@@ -5,20 +5,51 @@ layout: page
 
 This document describes which versions of Contour are supported by the Contour team.
 
-## Stable release
+## Supported releases
 
-Only the latest stable release is supported.
+Contour is changing both to quarterly releases and three supported releases.
 
+The first Contour version covered by the quarterly release cadence will be Contour v1.20, scheduled for late October 2021.
+
+At the time it is released, it will be the only supported version, and versions 1.21 and 1.22 will continue supporting back to Contour 1.20.
+
+When Contour 1.23 releases (nine months later), Contour 1.20 will fall out of support.
+
+The following table illustrates how this will work. The given dates are estimates, not guarantees.
+They are our best guess as to when each version will be released.
+
+| Version |v1.19 |v1.20|v1.21|v1.22|v1.23|
+|---------|--------|-------|-------|-------|-------|
+|Q3 2021 (September 2021) | :heavy_check_mark: |
+|Q4 2021 (October 2021) | :negative_squared_cross_mark: | :heavy_check_mark: |
+|Q1 2022 (January 2021) | :negative_squared_cross_mark: | :heavy_check_mark: |:heavy_check_mark: |
+|Q2 2022 (April 2021) | :negative_squared_cross_mark: | :heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: |
+|Q3 2022 (July 2021) | :negative_squared_cross_mark: | :negative_squared_cross_mark: |:heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: |
+
+## What does a release being "supported" mean?
+
+In short, "supported" means that Contour will issue fixes for security and other critical bugs for that release's supported lifetime.
+
+However, the project will require users to upgrade to the most recent patch release for a version to be supported.
+
+That is:
+- The latest patch version in each release is the supported version.
+- If you are not running the supported version from your release train, you'll need to upgrade first if you have any problems.
+- When a new patch is cut, that will become the supported version for that release.
+
+So, if Contour 1.20.0 is the only supported version, and Contour 1.20.1 is released, then the supported version will change to 1.20.1.
+
+If, later on, the supported versions are 1.21, 1.22, and 1.23, and 1.21.1, 1.22.1, and 1.23.1 are released, then the patch releases will be the only supported versions.
```

Comment:
so 1.21.0, 1.22.0, 1.23.0 will fall out of support?

Reply:
Yes. Is this not a standard feature of a patch release?

Reply:
In thinking about this again, I don't think there's a difference btw the two so my question is moot I suppose.  The question I had in mind originally is if our stance is such that 1.20.5 comes out and renders anything <= 1.20.4 out of support, then what happens if another vulnerability is found on, say, 1.20.2. Presumably that same vulnerability still exists on the latest 1.20.5 because the delta btw 1.20.5 and the previous 1.20.4 is precisely some patched vulnerability.  So we'd be forced to release a 1.20.6.

Now if for some reason 1.20.5 was not prone to this vulnerability, I think we'd be fine recommending users to just upgrade to 1.20.5.

Reply:
Yes, that's how I expected everything to work too.